s3_website.yml

```diff
@@ -44,30 +44,63 @@ cloudfront_distribution_config:
 # concurrency_level: 5
 
 redirects:
-  authentication/index.html: /v1/authentication/
-  certificates/index.html: /v1/certificates/
-  contacts/index.html: /v1/contacts/
-  domains/autorenewal/index.html: /v1/domains/autorenewal/
-  domains/forwards/index.html: /v1/domains/forwards/
-  domains/privacy/index.html: /v1/domains/privacy/
-  domains/records/index.html: /v1/domains/records/
-  domains/sharing/index.html: /v1/domains/sharing/
-  domains/zones/index.html: /v1/domains/zones/
-  domains/index.html: /v1/domains/
-  nameservers/vanity-nameservers/index.html: /v1/nameservers/vanity-nameservers/
-  nameservers/index.html: /v1/nameservers/
-  overview/index.html: /v1/
-  planned-changes/index.html: /v1/planned-changes/index.html
-  registrar/extended-attributes/index.html: /v1/registrar/extended-attributes/
-  registrar/prices/index.html: /v1/registrar/prices/
-  registrar/index.html: /v1/registrar/
-  services/domains/index.html: /v1/services/domains/
-  services/index.html: /v1/services/
-  subscriptions/index.html: /v1/subscriptions/
-  templates/domains/index.html: /v1/templates/domains/
-  templates/records/index.html: /v1/templates/records/
-  templates/index.html: /v1/templates/
-  users/index.html: /v1/users/
+  138: /
+  authentication: /v2/#authentication
+  certificates: /v1/certificates/
+  contacts: /v2/contacts/
+  domains/autorenewal: /v2/registrar/auto-renewal/
+  domains/forwards: /v1/domains/forwards/
+  domains/privacy: /v1/domains/privacy/
+  domains/records: /v2/zones/records/
+  domains/registry: /v2/domains/
+  domains/sharing: /v1/domains/sharing/
+  domains/zones: /v2/zones/
+  domains: /v1/domains/
+  nameservers/vanity-nameservers: /v1/nameservers/vanity-nameservers/
+  nameservers: /v1/nameservers/
+  OptionsNote: /
```

Comment:
?

We never had this page.

Reply:
I got it from this list: https://docs.google.com/spreadsheets/d/1t8Qv6N-Iof-URt28N4cFOgABs5uHxMQwJMLpE4YeGhA/edit#gid=77343041

Which page specifically, `OptionsNote`?